src/cdk/a11y/focus-trap.md

```diff
@@ -0,0 +1,12 @@
+### FocusTrap
+The `cdkFocusTrap` directive traps Tab key focus within an element. This is intended to be used to
+create components like [modal dialogs](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal).
```

Comment:
Clarified as "This is intended to be used to create accessible experience for components like..."